dev-util/ccache/ccache-4.10.1.ebuild

```diff
@@ -73,6 +73,7 @@ PATCHES=(
 	"${FILESDIR}"/${PN}-3.5-nvcc-test.patch
 	"${FILESDIR}"/${PN}-4.0-objdump.patch
 	"${FILESDIR}"/${PN}-4.10-avoid-run-user.patch
+	"${FILESDIR}"/${P}-libfmt11.patch
```

Comment:
```suggestion
	"${FILESDIR}"/${PN}-4.10-libfmt11.patch
```

Or change the patch files name ccache-4.10.1-libfmt11.patch

Reply:
:disappointed: